guides/distributed_training.py

```diff
@@ -456,4 +456,5 @@ def run_training(epochs=1):
     https://www.tensorflow.org/api_docs/python/tf/distribute/MirroredStrategy)
 4. [MultiWorkerMirroredStrategy docs](
     https://www.tensorflow.org/api_docs/python/tf/distribute/experimental/MultiWorkerMirroredStrategy)
+5. [Distributed training in `tf.keras`](https://towardsdatascience.com/distributed-training-in-tf-keras-with-w-b-ccf021f9322e)
```

Comment:
Thank you for the PR. Please use the full title of the article here. The link text should differentiate this link from the others, and should answer: why should the reader click on it?

Reply:
Thanks for the suggestion, I updated it. The article shows the use of `MirroredStrategy` and also discusses about some important gotchas like learning rate tuning, data pipeline optimization. Should I change the title of the article to reflect on that?